app/javascript/packages/phone-input/index.ts

```diff
@@ -165,6 +165,11 @@ export class PhoneInputElement extends HTMLElement {
       iti.selectedFlag.removeAttribute('aria-owns');
     }
 
+    const country = iti.getSelectedCountryData();
+    if (country.iso2 && this.codeInput) {
+      this.codeInput.value = country.iso2.toUpperCase();
+    }
```

Comment:
Is it important that we avoid the `change` event that would be triggered by calling to `syncCountryChangeToCodeInput` instead?

```suggestion
    this.syncCountryChangeToCodeInput();
```

Reply:
Also, could we add some JavaScript test coverage for these changes?

Reply:
Yeah, I originally tried just calling `syncCountryChangeToCodeInput`, but I believe it would replace a blank value with "+1" initially, which looked strange.